Resources/Locale/en-US/_RMC14/xeno/xeno-abilities.ftl

```diff
@@ -114,6 +114,16 @@ cm-xeno-invisibility-expire = We feel our invisibility end!
 cm-xeno-ovipositor-attach = We start attaching to the ovipositor.
 cm-xeno-ovipositor-detach = We start detaching from the ovipositor.
 
+# Retrieve Egg
+cm-xeno-retrieve-egg-no-eggs = We don't have any eggs stored!
+cm-xeno-retrieve-egg-too-many-eggs = We don't have space to store any more eggs!
+cm-xeno-retrieve-egg-stash-egg = We put the egg into our internal storage.
```

Comment:
Generally carrier messages when getting or retreiving eggs/parasites are like this:
"We grab one of the facehugger in our storage. Now sheltering: [huggers_cur] / [huggers_max]."
They show you how much space you have left